app.rb

```diff
@@ -0,0 +1,39 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep.rb'
+require './lib/user.rb'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  enable :sessions
```

Comment:
session is used effectively in the route, but do you need it? could your user class keep track of who is logged in?